scm/src/suite_info.py

```diff
@@ -57,7 +61,19 @@ def timestep(self, value):
 suite_list.append(suite('HAFS_v0_hwrf',          'tracers_HAFS_v0_hwrf.txt',             'input_HAFS_v0_hwrf.nml',            600.0, 1800.0, False))
 suite_list.append(suite('HAFS_v0_hwrf_thompson', 'tracers_HAFS_v0_hwrf_thompson.txt',    'input_HAFS_v0_hwrf_thompson.nml',   600.0, 600.0 , False))
 suite_list.append(suite('SCM_GSD_v1nssl',        'tracers_gsd_nssl.txt',                 'input_GSD_v1nssl.nml',              600.0, 600.0 , False))
+suite_list.append(suite('SCM_GSD_v1',            'tracers_gsd.txt',                      'input_GSD_v1.nml',                  600.0, 600.0 , False))
 suite_list.append(suite('SCM_RRFS_v1nssl',       'tracers_RRFS_v1nssl_nohail_noccn.txt', 'input_RRFS_v1nssl_nohailnoccn.nml', 600.0, 600.0 , False))
-  
-  
+suite_list.append(suite('SCM_csawmg',            'tracers_csawmg.txt',                   'input_csawmg.nml',                  600.0, 1800.0, False))
+
+def main():
+    
+    #print supported suites separated by commas
+    suite_string = ''
+    for s in suite_list:
+        if s._supported:
+            suite_string += s._name + ',' + s._name + '_ps' + ','  
```

Comment:
@grantfirl 
This behaves differently than before.
Before all suites in "suite_list" were created, but now only supported suites in "suite_list" are created. 
Also, isn't this also trying to create a "_ps" counterpart suite for each supported suite?

Reply:
@dustinswales Yes, this works differently than before. Previously, the CMakeLists.txt file told cmake to create a list of all suites to compile no matter what. Now, which suites get compiled depends on whether -DCCPP_SUITES is given to cmake. The old behavior should be retrieved by specifying -DCCPP_SUITES=ALL. If one omits -DCCPP_SUITES, CMakeLists.txt now calls this script, which returns a string of suites to compile, specifically, those where the `supported` attribute is true.

Reply:
The number of suites has been growing, leading to really long compilation times. This is supposed to help that, and now it works more similarly to UFS too.

Reply:
@grantfirl
Thanks for the explanation. I will update the tutorial to reflect this change.

Reply:
Also, this isn't creating any SDFs. It's only building a list of existing SDFs to actually create caps for and compile.

Reply:
You are correct. I mean't "which SDFs are used"